src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -1710,13 +1759,21 @@ def _get_hyper_param_config(self):
         return hyper_param_config
 
     def _write_to_disk_operations(
-        self, train_metrics_dict: dict, validation_results_dict: dict, lr_dict: dict, inf_time: float, epoch: int, context: PhaseContext
+        self,
+        train_metrics_dict: dict,
+        validation_results_dict: dict,
+        test_metrics_dict: dict,
+        lr_dict: dict,
+        inf_time: float,
+        epoch: int,
+        context: PhaseContext,
     ):
         """Run the various logging operations, e.g.: log file, Tensorboard, save checkpoint etc."""
         result_dict = {
             "Inference Time": inf_time,
             **{f"Train_{k}": v for k, v in train_metrics_dict.items()},
             **{f"Valid_{k}": v for k, v in validation_results_dict.items()},
+            **{f"Test_{k.replace(' ', '_')}": v for k, v in test_metrics_dict.items()},
```

Comment:
Question: Are we ok with whitespace in the names of logged values (wandb, tb, ...) ?
And what about the prints, are we ok with having `dataset_name` including whitespace when printing each metric result in format `{dataset_name}:{metric_name}` ?

Reply:
I personally againts doing any sanitization of strings inside the SG trainer.
First and foremost: Why? 
If some logger does not support whitespaces or `@` symbol (All in all this is wierd, but ok) - the concrete logger class should take care of that. 
That's my 5 cents.